protocol-tests/src/test/java/fixtures/dpgcustomization/DpgCustomizationTests.java

```diff
@@ -94,6 +109,36 @@ public void getPages() {
         Assertions.assertEquals(ProductReceived.MODEL, modelList.get(0).getReceived());
     }
 
+    @Test
+    public void getPagesTracing() {
+        TracingValidationPolicy tracingValidationPolicy = new TracingValidationPolicy();
+        this.asyncClient = new DpgClientBuilder().addPolicy(tracingValidationPolicy).buildAsyncClient();
+
+        Tracer TRACER = configureJaegerExporter();
+
+        Span userParentSpan = TRACER.spanBuilder("user-parent-span").startSpan();
+
+        Context context = new Context(
+                com.azure.core.util.tracing.Tracer.PARENT_TRACE_CONTEXT_KEY,
+                io.opentelemetry.context.Context.current().with(userParentSpan));
+
+        RequestOptions requestOptions = new RequestOptions();
+        requestOptions.setContext(context);
+
+        PagedFlux<BinaryData> pagedFlux = asyncClient.getPages("model", requestOptions);
```

Comment:
Also test syncClient. They are different, hence focus is syncClient, not asyncClient.